clients/replication/replication.go

```diff
@@ -104,6 +104,29 @@ func (c Client) List(ctx context.Context, params *ListParams) error {
 	return c.printReplication(printOpts)
 }
 
+func (c Client) Delete(ctx context.Context, replicationID string) error {
+	connection, err := c.GetReplicationByID(ctx, replicationID).Execute()
+	if err != nil {
+		return fmt.Errorf("failed to delete replication stream %q: %w", replicationID, err)
+	}
+
+	req := c.DeleteReplicationByID(ctx, replicationID)
+
+	// send delete request
+	err = req.Execute()
+	if err != nil {
```

Comment:
Nitpick: could collapse from line 113 to here into a single line, nested inside the conditional.:
```go
if err := c.DeleteReplicationByID(ctx, replicationID).Execute(); err != nil {
    //stuff
}
```